src/renderers/shared/ReactDebugTool.js

```diff
@@ -244,11 +301,13 @@ var ReactDebugTool = {
   onBeginLifeCycleTimer(debugID, timerType) {
     checkDebugID(debugID);
     emitEvent('onBeginLifeCycleTimer', debugID, timerType);
+    markBegin(debugID, timerType);
```

Comment:
Yeah, I wrote this PR yesterday while waiting for the taxi so it wasn’t really clean 😄